eng/packages/http-client-csharp/generator/TestProjects/Local/Mgmt-TypeSpec/tspCodeModel.json

```diff
@@ -1976,9 +1976,9 @@
                 {
                  "$id": "256",
                  "kind": "property",
-                 "name": "principalId",
-                 "serializedName": "principalId",
-                 "doc": "The principal ID of the assigned identity.",
+                 "name": "clientId",
```

Comment:
Why did this change?

Reply:
Due to the order of serialization.
The content is the same, my guess is the implementation change in TCGC